### ISSUE TITLE
Seer does not launch in Directory Opus

### ISSUE THREAD

**Issue by Koolstr** [ccseer/Seer#17]
I specified the right path for dopusrt.exe, yet even after restarting both applications and even my PC, pressing the keyboard shortcut does not bring up/activate Seer, no matter what you specify the keyboard shortcut to be.

In Directory Opus, if I hit the keyboard shortcut, the mouse very briefly gets the spinning wheel, so clearly something happens behind the scenes when I press the keyboard shortcut, but for some reason it is not showing anything.

Please look into this.

Also, as a side note, Seer has 'Run At System Start' enabled but it does not start on boot. Is this a local issue with my PC, or is there something wrong with Seer?

**Comment by ccseer** [comment]
Would you please try to set the option 'Track selected file in Explorer' off?

**Comment by Koolstr** [comment]
That didn't work still. Same thing happens. Any other ideas?

**Comment by Koolstr** [comment]
Another issue I am having is that Seer refuses to start on Windows boot, even with its shortcut being manually placed in the Startup folder. What's up with that?

**Comment by Koolstr** [comment]
It's been a while, no response, and the issue is still open. I'd just like to know, are you working on fixing this?

**Comment by ccseer** [comment]
I'm sorry.
I left for a trip. And I just got home a few days ago. A lot of work is
waiting on me in company.

And I had tried to work this out before the trip, but my DO is expired. So
I need more time.

THIS BUG WILL BE FIXED.

I'm sorry, I did reply to you at the first place.

**Comment by ccseer** [comment]
Would you please roll back to Seer-0.7.3?
I think the older version is compatible with DO.

**Comment by Koolstr** [comment]
It's all right, life happens. Take your time fixing it; this may make your application unusable for me as of now, but it is not the end of the world. Thank you for working on it.

I downgraded to 0.7.3 and it still does not work.

**Comment by Koolstr** [comment]
Two months have passed. Status on your progress on fixing this bug? I hope you did not just forget :P

**Comment by ccseer** [comment]
Hi~

this bug is already included in my TODO list.
But v0.8.0 is focus on UI.

My plan is to fix all the known bugs in v0.9.0.
I hope that would be ok.

**Comment by Koolstr** [comment]
That's fine. Just know that means you'll be having one less user until it is fixed >.<

**Comment by Caf** [comment]
Can't wait for that fix too! :)

**Comment by ccseer** [comment]
which version are you guys using ?  

Works really well here.

![image](https://cloud.githubusercontent.com/assets/15963166/18123108/f7035df2-6f9e-11e6-99c2-22271b79e559.png)

**Comment by ccseer** [comment]
![image](https://cloud.githubusercontent.com/assets/15963166/18123146/27e0b6ea-6f9f-11e6-8c92-f553efddce85.png)

**Comment by Koolstr** [comment]
Oddly enough, I am using a newer version of DOpus than you, but the problem still persists. This is true even after a few full clean system resets I had to perform.
![2016-08-31 09_53_13-about directory opus 11](https://cloud.githubusercontent.com/assets/2044760/18131680/b90a3f26-6f61-11e6-9ab3-9157eacbcecb.png)

Thanks for bringing focus back to this problem. Anyway, what version of Seer are you running? The UI is clearly improved. Is this the new UI redesign you said you were going to be making? When is the expected release date?

**Comment by Koolstr** [comment]
Also, you may want to look into why your settings window technically expands beyond its borders, as visible in the screenshot above, which was taken with Greenshot's window capture mode, as was the DOpus About splash screen. Probably connected to this issue as well, your settings window does not snap to screen borders or window edges, like nearly every other windows app.

**Comment by ccseer** [comment]
is it possible for you to move DO to a folder which dopusrt.exe path doesn't contain a Space ?

**Comment by ccseer** [comment]
you can try to check this update half hour later.

**Comment by Koolstr** [comment]
> you can try to check this update half hour later.

What do you mean by this?

**Comment by ccseer** [comment]
OK....

0.8.0 is available now...

**Comment by Koolstr** [comment]
Wow, that was quick! I'm liking the new UI, good job :)

I'll try your no-space approach and get back to you.

**Comment by Koolstr** [comment]
Didn't seem to help at all/change anything... :/

Interesting question: Why do you ask for the path to dopusrt.exe, the helper application, when the main app actually runs off of dopus.exe? Trying to run dopusrt.exe doesn't do anything..

**Comment by ccseer** [comment]
Seer has different window shadow and title bar, that's why the issues about window edge happen.

I asked the  DO team, they told me the only way to get selected file in DO is to use dopusrt.exe.
So evertime Seer tries to get the file path, it invokes the dopusrt.exe with a few paremeters. Then dopusrt.exe provides the information Seer wanted.

**Comment by Koolstr** [comment]
I'll take that means you have no intention of normalizing Seer's shadows and title bar, huh?

All right. Just wanted to make sure it wasn't a mistake.

So do you still intend to look into this issue to try to resolve it?

**Comment by Caf** [comment]
The new version with the path to the .exe does work for me. Thanks a lot. :+1:

**Comment by Koolstr** [comment]
What in the world, it works for you now but not me? What is your setup like, what versions are you running? And clarify what you mean path to the .exe please.

**Comment by Caf** [comment]
Nothing fancy, latest Win10 patched (64 bits) + latest DOPUS (not beta) and latest Seer. I'm on my Mac, can't check the version now sorry. Path to DOPUS.exe, like in the screenshot of this thread. And for the record, my path does have a space in one of the directory's name.

**Comment by Koolstr** [comment]
Please do get back to me with your DOpus and Win10 version numbers :)

**Comment by Caf** [comment]
Dopus : 11.19 x64 / Build 6005
Win 10 is v1607 (build 14393.105) (latest one / patch)

**Comment by ccseer** [comment]
A new support in Seer-1.5.0.

**Comment by jtourlamain** [comment]
I'm trying out seer 1.6.1 with DO 12, but I can't get it working. The directory opus 3rd party is pointing to C:\Program Files\GPSoftware\Directory Opus\dopusrt.exe When working in DO and pressing spacebar on a file, nothing happens

**Comment by ccseer** [comment]
Did you try a restart or something ?

or you can roll back:
https://bitbucket.org/coooorey/seer-download/downloads/Seer-1.4.0.exe

Jan Tourlamain <notifications@github.com>于2017年9月14日周四 下午8:38写道：

**Comment by jtourlamain** [comment]
ah, a full restart did the trick for 1.6.1 . Sorry I didn't try it before asking. I'm not used to restart anymore...
